dist/angular-patternfly.js

```diff
@@ -1465,6 +1466,209 @@ angular.module('patternfly.charts').directive('pfHeatmap', ["$compile", function
+ *                           return HTML markup for the tooltip contents. Setting this overrides the tooltipType value.
+ * <li>.area               - (optional) overrides the default Area properties of the C3 chart
+ * <li>.size               - (optional) overrides the default Size properties of the C3 chart
+ * <li>.axis               - (optional) overrides the default Axis properties of the C3 chart
+ * <li>.color              - (optional) overrides the default Color properties of the C3 chart
+ * <li>.legend             - (optional) overrides the default Legend properties of the C3 chart
+ * <li>.point              - (optional) overrides the default Point properties of the C3 chart
+ * </ul>
+ *
+ * @param {object} chartData the data to be shown as an area chart<br/>
+ * First and second Array elements, xData and yData, must exist, next data arrays are optional.<br/>
+ * <ul style='list-style-type: none'>
+ * <li>.xData      - Array, X values for the data points, first element must be the name of the data
+ * <li>.yData      - Array, Y Values for the data points, first element must be the name of the data
+ * <li>.yData1     - Array, Y Values for the data points, first element must be the name of the data
+ * <li>.[...]      - Array, Y Values for the data points, first element must be the name of the data
+ * </ul>
+ *
+ * @param {boolean=} showXAxis override config settings for showing the X Axis
+ * @param {boolean=} showYAxis override config settings for showing the Y Axis
+
+ * @example
+ <example module="patternfly.charts">
+   <file name="index.html">
+     <div ng-controller="ChartCtrl" class="row" style="display:inline-block; width: 100%;">
+       <div class="col-md-12">
+         <div pf-line-chart config="config" chart-data="data" chart-height="custChartHeight" show-x-axis="custShowXAxis" show-y-axis="custShowYAxis"></div>
+       </div>
+       <hr class="col-md-12">
+       <div class="col-md-12">
+         <div class="row">
+           <div class="col-md-6">
+             <form role="form"">
+               <div class="form-group">
+                 <label class="checkbox-inline">
+                   <input type="checkbox" ng-model="custShowXAxis">X Axis</input>
+                 </label>
+                 <label class="checkbox-inline">
+                   <input type="checkbox" ng-model="custShowYAxis">Y Axis</input>
+                 </label>
+               </div>
+             </form>
+           </div>
+           <div class="col-md-3">
+                 <button ng-click="addDataPoint()">Add Data Point</button>
+           </div>
+         </div>
+       </div>
+     </div>
+   </file>
+
+   <file name="script.js">
+     angular.module( 'patternfly.charts' ).controller( 'ChartCtrl', function( $scope ) {
+
+       $scope.config = {
+         chartId: 'exampleLine',
+         grid: {y: {show: false}},
+         point: {r: 1},
+         color: {pattern: ["#0033CF", "#00CF33"]}
```

Comment:
Please choose colors from the patternfly color palette found at https://www.patternfly.org/styles/color-palette/